### DISCUSSION TITLE
how can i configure the git 'GenericContribute' permission to perform an action?

### DISCUSSION THREAD

**Discussion by Geanne1999** [renovatebot/renovate#17480] [Request Help]
Message Error: You need the Git 'GenericContribute' permission to perform this action in Azure DevOps, but i already have the permissions to contribute. It worked in another project, but not in this one. Somebody has faced the same problem and found a solution? Thanks in advance.

**Comment by PhilipAbed** [comment]
i believe this is not related to renovate,

maybe check this: https://developercommunity.visualstudio.com/t/help-i-cannot-post-a-new-problem/382793
or look for a suitable answer that works for you on google.

hope that helps!
